src/event-manager.js

```diff
@@ -104,9 +106,10 @@ class DelegateHandlerEntry {
   }
 
   decrement() {
-    this.count--;
-
     if (this.count === 0) {
+      emLogger.warn('The same EventListener was disposed multiple times.');
+    }
+    if (--this.count === 0) {
```

Comment:
This and the block following it don't seem to guard against over disposal anymore, is this intentional?

Reply:
I think I missed the else part (-: